### PR TITLE
Add rainbow lights and brightness setting to webpage.

### PR DESCRIPTION
**Pull request by lauracowen**

### CODE REVIEW

File: disco-hat/disco-hat.ino [hunk 1]
```diff
@@ -34,22 +38,47 @@ void handleRoot() {
 void handleLedOn() {
   Serial.println("handleOn");
   led_state = ON;
-  strip.setPixelColor(0, strip.Color(255, 255, 255));
-  strip.show();
+//  strip.setPixelColor(0, strip.Color(255, 255, 255));
```
Comment (fraz3alpha) [inline disco-hat/disco-hat.ino]: Lets just remove these lines, rather than leave them commented out

File: disco-hat/disco-hat.ino [hunk 2]
```diff
@@ -34,22 +38,47 @@ void handleRoot() {
...
+//  strip.show();
   webServer.send(200, "text/html", responseHTML);
 }
 
 void handleLedOff() {
   Serial.println("handleOff");
   led_state = OFF;
-  strip.setPixelColor(0, strip.Color(0, 0, 0));
-  strip.show();
+//  strip.setPixelColor(0, strip.Color(0, 0, 0));
```
Comment (fraz3alpha) [inline disco-hat/disco-hat.ino]: Same here, lets delete these

File: disco-hat/disco-hat.ino [hunk 3]
```diff
@@ -64,6 +93,10 @@ void setup() {
   webServer.on("/", handleRoot);
   webServer.on("/on", handleLedOn);
   webServer.on("/off", handleLedOff);
+    webServer.on("/dim", handleLedDim);
```
Comment (fraz3alpha) [inline disco-hat/disco-hat.ino]: Can you make the indentation consistent here please

File: disco-hat/disco-hat.ino [hunk 4]
```diff
@@ -74,8 +107,45 @@ void setup() {
   webServer.begin();
 }
 
+    uint8_t j;
```
Comment (fraz3alpha) [inline disco-hat/disco-hat.ino]: Shift these left to line up

File: disco-hat/disco-hat.ino [hunk 5]
```diff
@@ -74,8 +107,45 @@ void setup() {
...
+    uint16_t i;
+
+unsigned long last_changed = 0;
+
 void loop() {
   dnsServer.processNextRequest();
   webServer.handleClient();
-  Serial.println(led_state);
+  //Serial.println(led_state);
```
Comment (fraz3alpha) [inline disco-hat/disco-hat.ino]: Just remove this line

**Review by fraz3alpha — CHANGES_REQUESTED** [review]
Just a few things to tidy up, but I'm happy to accept this :)

**Review by lauracowen — COMMENTED** [review]
Done all changes as requested.

**Review by fraz3alpha — APPROVED** [review]

**Pull request merged**